aiida-registry-app/src/Components/utils.js

```diff
@@ -0,0 +1,10 @@
+export function extractSentenceAroundKeyword(sentence, keyword) {
+    const regex = new RegExp(`"([^"]*${keyword}[^"]*)"`, 'gi');
```

Comment:
Not sure I understand the regex here. It tries to get the sentence inside a pair of double quotes?

Reply:
Exactly, I noticed that this method has some problems as it matches the entire object. So if you search for 'description' it will match and display 'description'. Maybe looping through the object keys will be better than this, will try it.

Reply:
Thanks for the change!